data/856/875/95/85687595.geojson

```diff
@@ -11,14 +11,23 @@
     "geom:longitude":26.087654,
     "iso:country":"RO",
     "label:eng_x_preferred_longname":[
-        "Bucure\u015fti City"
+        "Bucharest"
```

Comment:
Since this is a unitary authority (capital city), I'm proposing this to simply be "Bucharest".. "Bucharest City" is redundant.